plugins/samson_ledger/.gitignore

```diff
@@ -0,0 +1,8 @@
+.bundle/
```

Comment:
there is no dummy app here ... and log / .bundle should already be in base samson